investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java

```diff
@@ -51,13 +51,13 @@ public class ProcessInvestigationDataUtil {
     @Transactional
     public InvestigationTransformed transformInvestigationData(Investigation investigation) {
 
-        InvestigationTransformed investigationTransformed = new InvestigationTransformed();
+        InvestigationTransformed investigationTransformed = new InvestigationTransformed(investigation.getPublicHealthCaseUid());
 
         transformPersonParticipations(investigation.getPersonParticipations(), investigationTransformed);
         transformOrganizationParticipations(investigation.getOrganizationParticipations(), investigationTransformed);
         transformActIds(investigation.getActIds(), investigationTransformed);
         transformObservationIds(investigation.getObservationNotificationIds(), investigationTransformed);
-        transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod());
+        transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod(), investigationTransformed);
```

Comment:
Does this have a null check?

Reply:
Line 54 might already have initialized the object and we might not need a NPE check here..

Reply:
Every nested data is checked inside the corresponded method.